exercises/practice/binary/binary_test.rb

```diff
@@ -3,42 +3,42 @@
 
 # Common test data version: 1.1.0 dd3c07e
 class BinaryTest < Minitest::Test
-  def test_binary_0_is_decimal_0
+  def test_binary_0_is_decimal_0 # rubocop:disable Naming/VariableNumber
```

Comment:
Is it possible to have Rubocop disable this rule for this file only?

Reply:
It might be better to reword the titles of test methods as well.